### PR TITLE
feat: add  background page

### PR DESCRIPTION
**Pull request by lelenei**
Closes mf-702

### CODE REVIEW

File: packages/mask/src/plugins/Avatar/Application/NFTListPage.tsx [hunk 1]
```diff
@@ -141,29 +150,6 @@ export function NFTImageCollectibleAvatar({
...
-        <img
-            className={classes.image}
-            src={
-                theme.palette.mode === 'dark'
-                    ? assetPlayerFallbackImageDark.toString()
-                    : assetPlayerFallbackImageLight.toString()
-            }
-        />
-    )
+    return
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Avatar/Application/NFTListPage.tsx]: Why not remove it?

File: packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx [hunk 2]
```diff
@@ -98,10 +98,12 @@ export const PluginVerifiedWalletStatusBar = memo<PluginVerifiedWalletStatusBarP
                 providerIcon: account ? providerDescriptor?.icon : undefined,
                 iconFilterColor: account ? providerDescriptor?.iconFilterColor : '',
                 formattedAddress: Others?.formatAddress(account || (defaultVerifiedWallet?.identity ?? ''), 4),
-                addressLink: Others?.explorerResolver.addressLink?.(
-                    account ? chainId : defaultChainId,
-                    account || (defaultVerifiedWallet?.identity ?? ''),
-                ),
+                addressLink:
```
Comment (nuanyang233) [inline packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx:101]: Please revert this change. `addressLink` should be a URL and should not use account.

**Review by nuanyang233 — CHANGES_REQUESTED** [review]

File: packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx (hunk 2 above)
Comment (lelenei) [reply packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx:101]: you tested?

Comment (nuanyang233) [reply packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx:101]:
```
 <Link
          href={addressLink}
          target="_blank"
          title="View on Explorer"
          rel="noopener noreferrer"
          onClick={(event) => {
                 event.stopPropagation()
           }}
          className={classes.linkIcon}>
         <Icons.LinkOut size={14} className={classes.linkIcon} />
</Link>
```

So you want to set an address to `href`? Do you think that makes sense?

Comment (nuanyang233) [reply packages/mask/src/utils/components/WalletStatusBar/PluginVerifiedWalletStatusBar.tsx:101]: And test result: 
image

image

**Review by guanbinrui — APPROVED** [review]
